Gopkg.toml

```diff
@@ -69,3 +69,6 @@ required = [
   name = "upper.io/db.v3"
   version ="3.5.7"
 
+[[constraint]]
+  name = "github.com/aliyun/aliyun-oss-go-sdk"
```

Comment:
Apache v2 is an acceptable licence.

Reply:
yes it is. https://github.com/aliyun/aliyun-oss-go-sdk#license